controls/mysql_conf.rb

```diff
@@ -67,7 +67,7 @@
 control 'mysql-conf-02' do
   impact 0.5
   title 'only one instance of mysql should run on a server'
-  describe command('ps aux | grep mysqld | egrep -v "grep|mysqld_safe|logger" | wc -l') do
+  describe command("ps aux | grep #{service_name} | egrep -v \"grep|mysqld_safe|logger\" | wc -l") do
```

Comment:
I always hate to see `ps | ... | grep -v grep`, could we switch to `pgrep` here? (it even has a count function)